google/resource_sql_database_instance.go

```diff
@@ -56,6 +56,13 @@ func resourceSqlDatabaseInstance() *schema.Resource {
 							Optional: true,
 							Elem:     &schema.Schema{Type: schema.TypeString},
 						},
+						"availability_type": &schema.Schema{
+							Type:             schema.TypeString,
+							Optional:         true,
+							DiffSuppressFunc: suppressFirstGen,
+							// Set computed instead of default because this property is for second-gen only.
+							Computed: true,
```

Comment:
Likewise, can you just confirm for me that the value is actually Computed (i.e. there's a server-side default that will be returned if this is unset)?